Src/PerformanceCollector/Shared/QuickPulseTelemetryModule.cs

```diff
@@ -406,6 +428,33 @@ private void CollectionThreadWorker(object state)
                 Thread.Sleep(timeLeftUntilNextTick > TimeSpan.Zero ? timeLeftUntilNextTick : TimeSpan.Zero);
             }
         }
+        
+
+        private void InitializeCollectionThread()
+        {
+            try
+            {
+                this.topCpuCollector.Initialize();
+            }
+            catch (Exception e)
+            {
+                // whatever happened, don't bring the thread down
+                QuickPulseEventSource.Log.UnknownErrorEvent(e.ToInvariantString());
+            }
+        }
+
+        private void CloseCollectionThread()
+        {
+            try
+            {
+                this.topCpuCollector.Close();
```

Comment:
So based on Initialize really being more like "Start" I would suggest that maybe this should be "Stop" or something similar.